posthog/temporal/batch_exports/sql.py

```diff
@@ -98,39 +98,82 @@
     """
 SELECT
     $fields
-FROM
-    events_batch_export(
-        team_id={team_id},
-        lookback_days={lookback_days},
-        interval_start={interval_start},
-        interval_end={interval_end},
-        include_events={include_events}::Array(String),
-        exclude_events={exclude_events}::Array(String)
-    ) AS events
+FROM (
```

Comment:
guess we can clean up the views in a follow up PR?

Reply:
Yup!